### DISCUSSION TITLE
Does this tool now support mapping original GPS exif data?

### DISCUSSION THREAD

**Discussion by michalecm** [TheLastGimbus/GooglePhotosTakeoutHelper#235] [General]
Does this tool now support mapping original GPS exif data?

**Comment by TheLastGimbus** [comment]
Moved this to #236 

tl;dr sorry but still no :( it does not *move* gps data from jsons to exif, if you edited it in gphotos. BUT, your photos should have their exifs as original, so this shouldn't be a big problem...
